modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -147,37 +211,39 @@ object FormInputEV {
         }
         .orElse(validationError)
 
-      FormInput(
-        p.action,
-        p.actionPosition,
-        p.as,
-        p.className,
-        p.clazz,
-        p.content,
-        p.control,
-        p.disabled,
-        error,
-        p.fluid,
-        p.focus,
-        p.icon,
-        p.iconPosition,
-        p.inline,
-        p.input,
-        p.inverted,
-        p.label,
-        p.labelPosition,
-        p.loading,
-        js.undefined,
-        onTextChange(p),
-        p.required,
-        p.size,
-        p.tabIndex,
-        p.tpe,
-        p.transparent,
-        p.width,
-        s.curValue
-      )(
-        (p.modifiers :+ (^.id := p.id) :+ (^.onBlur --> onBlur(p, s)): _*)
+      OuterDiv.withRef(outerRef)(
```

Comment:
Are we wrapping then the inputs in a `div`. I guess that will require a css change in explore

Reply:
It's possible we'll need changes, but I don't think so. I'm not aware of any direct descendent CSS to the field, but there may be some.